change-validation/src/test/scala/com/normation/plugins/changevalidation/api/ChangeRequestApiTest.scala

```diff
@@ -0,0 +1,659 @@
+            )
+          ),
+          Map.empty,
+          Map.empty
+        ),
+        ConfigurationChangeRequest(
+          ChangeRequestId(6),
+          None,
+          ChangeRequestInfo("third cr group", "My group third change"),
+          Map.empty,
+          Map(
+            NodeGroupId(NodeGroupUid("group_001")) -> NodeGroupChanges(
+              NodeGroupChange(
+                Some(mockNodeGroups.g0),
+                NodeGroupChangeItem(
+                  actor,
+                  DateTime.parse("2023-06-06T00:00:00.000Z"),
+                  Some("group_001 delete change reason"),
+                  DeleteNodeGroupDiff(mockNodeGroups.g0)
+                ),
+                List.empty
+              ),
+              List.empty
+            )
+          ),
+          Map.empty,
+          Map.empty
+        )
+      ),
+      Deployed.id   -> List(
+        ConfigurationChangeRequest(
+          ChangeRequestId(7),
+          None,
+          ChangeRequestInfo("first cr rule", "My rule first change"),
+          Map.empty,
+          Map.empty,
+          Map(
+            RuleId(RuleUid("rule_001")) -> RuleChanges(
+              RuleChange(
+                None,
+                RuleChangeItem(
+                  actor,
+                  DateTime.parse("2023-07-07T00:00:00.000Z"),
+                  Some("rule_001 change reason"),
+                  AddRuleDiff(mockRules.rules.rpmRule.copy(tags = Tags.fromMaps(List(Map("key" -> "value")))))
+                ),
+                List.empty
+              ),
+              List.empty
+            )
+          ),
+          Map.empty
+        ),
+        ConfigurationChangeRequest(
+          ChangeRequestId(8),
+          None,
+          ChangeRequestInfo("second cr rule", "My rule second change"),
+          Map.empty,
+          Map.empty,
+          Map(
+            // To this point, RuleChange has a different 'change' implementation than others : it only takes 'firstChange'
+            RuleId(RuleUid("rule_001")) -> RuleChanges(
+              RuleChange(
+                Some(mockRules.rules.rpmRule),
+                RuleChangeItem(
+                  actor,
+                  DateTime.parse("2023-08-08T00:00:00.000Z"),
+                  Some("rule_001 change reason"),
+                  ModifyToRuleDiff(
+                    mockRules.rules.rpmRule.copy(
+                      name = "rule_002",
+                      shortDescription = "testing rule change",
+                      longDescription = "testing rule change",
+                      directiveIds = Set(DirectiveId(DirectiveUid("directive1"))),
+                      targets = Set(AllTargetExceptPolicyServers),
+                      isEnabledStatus = false,
+                      isSystem = false
+                      // ignored changes :
+                      // categoryId = RuleCategoryId("rule_category_001"),
+                      // policyMode = Some(PolicyMode.Audit),
```

Comment:
same: I don't see why these change are not tracked. PolicyMode at least has clear impact on prod. 
Can you look to that and potentially open a ticket in the correct branch if needed ?

Reply:
Same answer too, the diff is actually taken into account, here the previous version did not 'serialize' the change for these fields, so I kept that behavior, but it could be changed without risk

Reply:
Cf. the comment above (I think I would be better off in another PR)